src/main/jsx/job/dependencyGraph.jsx

```diff
@@ -30,20 +29,46 @@ module.exports = React.createClass({
   },
   componentDidMount: function() {
     window.addEventListener('resize', this.handleResize);
-    this.setState({
-      job: this.props.job,
+
+    var stateObject = {
+      nodeTree: ReactDOM.findDOMNode(this.refs.nodeTree),
+      nodeTreeGroup: ReactDOM.findDOMNode(this.refs.nodeTreeGroup),
```

Comment:
The DOM nodes should not be included in the `state`.
They are referable by using `refs`.